resources/scripts/dag-user-sync.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Check if the correct number of arguments is provided
+if [ "$#" -ne 2 ]; then
+    echo "Usage: $0 <source_server> <target_server>"
+    exit 1
+fi
+
+# Server addresses from arguments
+SOURCE_SERVER="$1"
+TARGET_SERVER="$2"
+
+# Fixed paths
+SOURCE_PATH="/beta-rewards"
+TARGET_PATH="/add-participant"
+
+# Construct URLs
+SOURCE_URL="http://$SOURCE_SERVER$SOURCE_PATH"
+TARGET_URL="http://$TARGET_SERVER$TARGET_PATH"
+
+# Fetch the JSON data from the source URL
+echo "Fetching data from $SOURCE_URL"
+json_data=$(curl -s $SOURCE_URL)
+
+# Parse the JSON data to extract names, skipping unknown participants
```

Comment:
Same.

Reply:
They'd show up again as soon as found on the new dag, so no need.

It's only the value for decryption that we need, otherwise it defaults to unknown participants (and I feat adding them is probably more prone to errors in this script, with braces, quotes etc. )


I'm not sure. This was supposed to be a quick fix for dag transition, whereas now we'd have the service 🤔 .

Reply:
There was a manual process to fill the missing users in the new DAG.